packages/hdwallet-keepkey/src/ethereum.ts

```diff
@@ -39,178 +39,168 @@ import {
 
 import * as EIP55 from 'eip55'
 
- /**
-  * Mixin Constructor that adds ETH support to a KeepKeyHDWallet
-  */
-export function KeepKeyETHWallet<TBase extends Constructor>(Base: TBase) {
-  return class KeepKeyETHWallet extends Base implements ETHWallet {
-    _supportsETH: boolean = true
-    transport: KeepKeyTransport
+export async function keepkey_ethSupportsNetwork (chain_id: number): Promise<boolean> {
```

Comment:
@keepkeyjon can we drop the `keepkey_` prefix? When importing you can do `import { * as Something}` and `Something. ethSupportsNetwork`

Reply:
sure